packaging/debian/control

```diff
@@ -47,14 +49,7 @@ Build-Depends: debhelper (>= 9),
     asciidoc-base | asciidoc, xmlto,
     dh-autoreconf
 
-Package: libfty-discovery0
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: fty-discovery shared library
- This package contains shared library for fty-discovery:
- 42ity service for discovering devices
-
-Package: libfty-discovery-dev
+Package: fty-discovery
 Architecture: any
 Section: libdevel
```

Comment:
extraneous

Reply:
We don't need anymore the library, it was used only by the discovery package. Is it bad renaming?